lxd/auth/driver_rbac.go

```diff
@@ -474,7 +474,12 @@ func (r *rbac) syncPermissions(username string) error {
 	return nil
 }
 
+<<<<<<< HEAD:lxd/auth/driver_rbac.go
 func (r *rbac) postResources(updates []rbacResource, removals []string, force bool) error {
+=======
+// Sends resource updates and removals to the API server and handles conflicts by syncing projects.
+func (r *Server) postResources(updates []rbacResource, removals []string, force bool) error {
```

Comment:
@monstermunchkin  Due to this function, I added another commit. Even after rebasing, it was still giving me an error. I want you to remove this function and add `func (r *rbac) postResources(updates []rbacResource, removals []string, force bool) error {}` that is matched with the LXD updated code.

Reply:
You'll need to manually fix this change.